README.wcf.md

```diff
@@ -0,0 +1,94 @@
+# Featured Tags
+
+* `4.8`
+  * `docker pull mcr.microsoft.com/dotnet/framework/wcf:4.8`
+
+## About This Image
+The Windows Communication Foundation (WCF) is a framework for building service-oriented applications. Using WCF, you can send data as asynchronous messages from one service endpoint to another. A service endpoint can be part of a continuously available service hosted by IIS, or it can be a service hosted in an application.
+
+# How to Use the Image
+
+The [WCF Docker samples](https://github.com/Microsoft/dotnet-framework-docker/tree/master/samples/wcfapp) show various ways to use WCF and Docker together.
+
+## Container sample: Run a WCF application
+You can quickly run a container with a pre-built [sample WCF Docker image](https://hub.docker.com/_/microsoft-dotnet-framework-samples/), based on the WCF Docker sample.
+
+Type the following [Docker](https://www.docker.com/products/docker) command to start a WCF service container:
+```console
+docker run --name wcfservicesample --rm -it mcr.microsoft.com/dotnet/framework/samples:wcfservice
+```
+Find the IP address of the container instance.
+```console
+docker inspect --format="{{.NetworkSettings.Networks.nat.IPAddress}}" wcfservicesample
+172.26.236.119
+```
+Type the following Docker command to start a WCF client container, set environment variable HOST to the IP address of the wcfservicesample container:
+```console
+docker run --name wcfclientsample --rm -it -e HOST=172.26.236.119 mcr.microsoft.com/dotnet/framework/samples:wcfclient
+```
+
+# Related Repos
+
+.NET Framework:
+
+* [dotnet/framework/runtime](https://hub.docker.com/_/microsoft-dotnet-framework-runtime/): .NET Framework Runtime
+* [dotnet/framework/aspnet](https://hub.docker.com/_/microsoft-dotnet-framework-aspnet/): ASP.NET Web Forms and MVC
+* [dotnet/framework/sdk](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/): .NET Framework SDK
+* [dotnet/framework/samples](https://hub.docker.com/_/microsoft-dotnet-framework-samples/): .NET Framework, ASP.NET and WCF Samples
+
+.NET Core:
+* [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/) for .NET Core images.
```

Comment:
(nit) should insert a space here to be consistent with the previous section.

Reply:
done